src/Assets/styleController.ts

```diff
@@ -43,6 +50,13 @@ const themes: IThemes = {
       'font-size': '16px',
       '--main-header-color': '#B85A27',
       '--midBlue-color': '#41528C',
+      '--option-card-background-color': '#FFFFFF',
```

Comment:
I don't think that we need to add anything that is just the color white to the style controller.